terraform/api.tf

```diff
@@ -16,12 +16,13 @@ resource "aws_lambda_function" "api" {
 
   environment {
     variables = {
-      NODE_ENV           = var.target_env
-      AWS_S3_REGION      = var.region
-      RUNTIME_ENV        = "hosted"
-      POSTGRES_HOST      = aws_rds_cluster.pgsql.endpoint
-      POSTGRES_DATABASE  = aws_rds_cluster.pgsql.database_name
-      POSTGRES_PASSWORD  = data.aws_ssm_parameter.postgres_password.value
+      NODE_ENV          = var.target_env
+      AWS_S3_REGION     = var.region
+      RUNTIME_ENV       = "hosted"
```

Comment:
what is the benefit of setting `hosted`?

Reply:
Not sure about that, I have added only the `export` secret.

Reply:
the diff is because of the reformat